addons/web/static/src/views/calendar/calendar_common/calendar_common_renderer.xml

```diff
@@ -10,7 +10,7 @@
                 <span class="d-none d-lg-inline fc-time" t-esc="startTime" />
                 <t> </t>
             </t>
-            <div class="o_event_title text-truncate" t-out="title"/>
+            <div class="o_event_title text-truncate text-nowrap" t-out="title"/>
```

Comment:
Not sure this one is necessary as `text-truncate` already adds a `white-space: nowrap;`

![Screenshot 2025-01-21 at 14 40 56](https://github.com/user-attachments/assets/c928c6a9-41f3-42b5-86bd-920434dc5076)